app/main.py

```diff
@@ -298,17 +298,18 @@ def setup_env(request):
     env.repo, env.action = get_repo_and_action(request)
     env.config = config.Configuration(env.repo or '*')
 
-    env.analytics_id = config.get('analytics_id')
-    env.amp_gtm_id = config.get('amp_gtm_id')
-    env.maps_api_key = config.get('maps_api_key')
+    env.analytics_id = env.config.get('analytics_id')
+    env.amp_gtm_id = env.config.get('amp_gtm_id')
+    env.maps_api_key = env.config.get('maps_api_key')
 
     # Internationalization-related stuff.
     env.charset = select_charset(request)
     env.lang = select_lang(request, env.config)
     env.rtl = env.lang in const.LANGUAGES_BIDI
 
     # Determine the resource bundle to use.
-    env.default_resource_bundle = config.get('default_resource_bundle', '1')
+    env.default_resource_bundle = (
+        env.config.get('default_resource_bundle') or '1')
```

Comment:
Can't you write env.config.get('default_resource_bundle', '1') here?

Reply:
Done.